experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/AggregationSelector.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getEnv } from '@opentelemetry/core';
+import { Aggregation, AggregationSelector, AggregationTemporality, AggregationTemporalitySelector, InstrumentType } from '@opentelemetry/sdk-metrics';
+
+/**
+ * Default {@link AggregationTemporalitySelector} when the OTLP Metrics Exporter is configured with environ `OTEL_METRICS_EXPORTER`.
+ */
+const CumulativeTemporalitySelector: AggregationTemporalitySelector = () => AggregationTemporality.CUMULATIVE;
+
+/**
+ * Default {@link AggregationSelector} when the OTLP Metrics Exporter is configured with environ `OTEL_METRICS_EXPORTER`.
```

Comment:
Well, not the "default" one for the SDK. It can be used as the default one for OTLP exporters when they are automatically configured. I'll update this description.

Reply:
thanks! :slightly_smiling_face: